src/main/scala/chisel3/ltl/LTL.scala

```diff
@@ -414,8 +415,10 @@ sealed abstract class AssertPropertyLike {
   )(
     implicit sourceInfo: SourceInfo
   ): Unit = {
-    val clocked = clock.fold(prop)(prop.clock(_))
-    createIntrinsic(label)(sourceInfo)(clocked.inner, disable.map(!_.value))
+    block(layers.Verification.Assert) {
+      val clocked = clock.fold(prop)(prop.clock(_))
+      createIntrinsic(label)(sourceInfo)(clocked.inner, disable.map(!_.value))
+    }
```

Comment:
I think this code path could create LTL assume/cover so I think they need to go to assume/cover layers.